src/util/func.cpp

```diff
@@ -202,6 +203,13 @@ std::string getFunctionFile(const faabric::Message& msg)
     return path.string();
 }
 
+std::string getEncryptedFunctionFile(const faabric::Message& msg)
+{
+    auto path = getFunctionDir(msg);
+    path.append(encryptedFuncFile);
+
+    return path.string();
+}
```

Comment:
Think there's a missing newline here between the functions, surprised formatting didn't catch